extra/release_tool.py

```diff
@@ -698,7 +698,10 @@ def do_list_repos(args, optional_too, only_backend):
             )
         )
     elif args.list_format == "json":
-        print(json.dumps(repos_versions_dict))
+        json_object = {"release": repos_versions_dict["integration"], "repos": []}
+        for name, version in repos_versions_dict.items():
+            json_object["repos"].append({"name": name, "version": version})
```

Comment:
```suggestion
        json_object = {"release": repos_versions_dict["integration"], "repos": list(map(lambda item:{item[0]:item[1]}, repos_versions_dict.items())) }
```

hmm... doesn't look toooo nice in python 🙈